configurations/create_initial_kenya_pool.py

```diff
@@ -7,7 +7,7 @@
 )
 
 PIPELINE_CONFIGURATION = PipelineConfiguration(
-    pipeline_name="Create-Kenya-Pool",
+    pipeline_name="Create-Initial-Kenya-Pool",
```

Comment:
Consider updating the archive bucket on L248 as well.

Reply:
Updated, thanks.